tests/DependencyInjection/XmlDoctrineExtensionTest.php

```diff
@@ -4,10 +4,12 @@
 
 namespace Doctrine\Bundle\DoctrineBundle\Tests\DependencyInjection;
 
+use PHPUnit\Framework\Attributes\IgnoreDeprecations;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
 
+#[IgnoreDeprecations]
```

Comment:
```suggestion
#[IgnoreDeprecations('Since symfony/dependency-injection 7.4: XML configuration format is deprecated, use YAML or PHP instead.')]
```

I would suggest to only ignore that specific deprecation? :thinking: I'm guessing its that message only, or?

Does this mean eventually (seems this 7.4 change is not merged up into 8.0 yet) we also need to exclude this test from running on Symfony 8?

Reply:
The `IgnoreDeprecations` constructor argument has been recently added, and is currently available only in development versions of phpunit. See https://github.com/doctrine/DoctrineBundle/actions/runs/18157826597